node/src/server-modules/GlideJson.ts

```diff
@@ -237,4 +237,75 @@ export class GlideJson {
 
         return _executeCommand<ReturnTypeJson<boolean>>(client, args);
     }
+
+    /**
+     * Deletes the JSON value at the specified `path` within the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) If `null`, deletes the entire JSON document at `key`.
+     * @returns - The number of elements removed. If `key` or `path` doesn't exist, returns 0.
+     *
+     * @example
+     * ```typescript
+     * console.log(await GlideJson.set(client, "doc", "$", '{a: 1, nested: {a:2, b:3}}'));
+     * // Output: "OK" - Indicates successful setting of the value at path '$' in the key stored at `doc`.
+     * console.log(await GlideJson.del(client, "doc", "$..a"));
+     * // Output: 2 - Indicates successful deletion of the specific values in the key stored at `doc`.
+     * console.log(await GlideJson.get(client, "doc", "$"));
+     * // Output: "[{nested: {b: 3}}]" - Returns the value at path '$' in the JSON document stored at `doc`.
+     * console.log(await GlideJson.del(client, "doc"));
+     * // Output: 1 - Deletes the entire JSON document stored at `doc`.
+     * ```
+     */
+    static async del(
+        client: BaseClient,
+        key: GlideString,
+        options?: { path: GlideString },
```

Comment:
```suggestion
        path?: GlideString,
```

Reply:
it was suggested to put path in an options object to keep consistency (https://github.com/valkey-io/valkey-glide/pull/2491#discussion_r1809688192; having just `path` is def more easy to use, what do we care about more?

Reply:
it's easier with just path.  I json.forgot that we decided to use the options objects throughout - which is easier to extend.  Just json.del my comment ;)